python/hsml/model_registry.py

```diff
@@ -19,10 +19,10 @@
 import humps
 from hsml import util
 from hsml.core import model_api
-from hsml.python import signature as python_signature  # noqa: F401
 from hsml.sklearn import signature as sklearn_signature  # noqa: F401
 from hsml.tensorflow import signature as tensorflow_signature  # noqa: F401
 from hsml.torch import signature as torch_signature  # noqa: F401
+from python.hsml.python import signature as python_signature  # noqa: F401
```

Comment:
why only change the parent module of this?

Reply:
This is a mistake, which is reversed in the next PR. https://github.com/logicalclocks/hopsworks-api/pull/228/commits/0e24b88a9025d3880c46618b133c20a956ed513e

The problem is that for some reason, if you start python from repo's root, hsml.python is available only as python.hsml.python. In all other places, hsml.python can be imported without any problems.

Reply:
can you cherry pick the commit to this PR